templates/settings.erb

```diff
@@ -220,7 +220,7 @@ puppetca_path: "<%= @puppetca_path %>"
 # installed is removed from the puppet master server if the puppet
 # master server is running on the same machine as cobbler. This
 # requires puppet_auto_setup above to be enabled
-remove_old_puppet_certs_automatically: <%= @remove_old_puppet_certs %>
+remove_old_puppet_certs_automatically: <%= @sign_puppet_certs_automatically %>
```

Comment:
This won't work cause the name of the param in `init.pp` isn't updated to match this value.